src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/ConceptsTestCases.java

```diff
@@ -27,8 +28,9 @@ public void testWithURL() {
 	}
 
 	private void testGetConcepts(String source, String... expectedConcepts) {
-		Concepts concepts = getConnector().concepts(source,
-				null, null, null, null, null, null, null);
+		ConceptsRequest cr = new ConceptsRequest();
```

Comment:
Please use a method like in CombineCall to generate the request, for example buildBasicRequest().
The results could look like:
`Concepts concepts = getConnector().concepts(buildBasicRequest());`